lambda-notes/src/components/notes-container/Allnotes.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import styled from 'styled-components'
+
+const NoteWrap = styled.div `
+width: 130px;
+height: 130px;
+display: flex;
+justify-content: flex-start;
+flex-direction: row;
+background-color: white;
+padding: 0px;
+`
+
+export default class AllNotes extends Component {
+  constructor(props) {
+    super(props);
+   // this.state = {
+     // notes: []
+    //};
+  }
+  
+
+ /*componentDidMount() {
+    axios
+      .get('https://fe-notes.herokuapp.com/note/get/all')
+      .then(response => {
+        this.setState(() => ({ notes: response.data }));
+      })
+      .catch(error => {
+        console.error('Server Error', error);
+      });
+ */
+
+  render() {
+    return (
+      <div className="notes-list">
+        {this.props.notes.map(note => (
+         <Link to={`/notes/${note._id}`} key={note._id}>
+          <SingleNoteDetails
+          key={note._id} 
```

Comment:
don't need a key on this, key is only needed on the top (root element) inside the `map` return